projects/plugins/boost/app/modules/optimizations/page-cache/pre-wordpress/Request.php

```diff
@@ -297,4 +301,33 @@ public function is_feed() {
 
 		return \is_feed();
 	}
+
+	/**
+	 * Return true if the Page Cache module is disabled.
+	 *
+	 * If Status and Page_Cache are not available, it means the plugin is not loaded.
+	 * This function will be called later when writing a cache file to disk.
+	 * It's then that we can check if the module is active.
+	 *
+	 * @return bool
+	 */
+	public function is_module_disabled() {
+
+		// A simple check to make sure we're in the output buffer callback.
+		if ( ! function_exists( '\is_feed' ) ) {
+			return false;
+		}
```

Comment:
I think it would be better to return `null` if we don't know yet. It's a falsy value so any callers which only care about a truthy value can just do a truthy check but the function doesn't lie about what it doesn't know yet :)

Reply:
Thanks, code has been updated.